examples/MOT17/multiedge_inference_bench/workspace/reid_job/rank/all_rank.csv

```diff
@@ -0,0 +1,2 @@
+time,paradigm,algorithm,basemodel,batch_size,mAP,rank_1,rank_2,rank_5,cmc
```

Comment:
Do not upload CSV to the KubeEdge Ianvs GitHub repository. The author can use Kaggle to keep the ianvs version of MOT17, the ranking CSV result, and the generated report, with a link provided in the readme in the ianvs repo. As for Kaggle, add kubeedge-ianvs as an admin for the dataset so members can find it easily. See [Kaggle link](https://www.kaggle.com/kubeedgeianvs/datasets)